### ISSUE TITLE
Fix typings for EventtStream

### ISSUE THREAD

**Issue by kanitw** [vega/vega-lite#4620]
Currently [it's `any`](https://github.com/vega/vega-lite/blob/master/src/vega.schema.ts#L54).

However, correctly importing it leads to the following error:

```
$ node --stack-size=4000 ./node_modules/.bin/ts-json-schema-generator --no-type-check --path tsconfig.json --type TopLevelSpec > build/vega-lite-schema.json && npm run renameschema && cp build/vega-lite-schema.json _data/
UnknownNodeError: Unknown node "GlobalEventHandlers" (ts.SyntaxKind = 165) at /Users/kanitw/Documents/code/vl/node_modules/typescript/lib/lib.dom.d.ts(4033,89)
```

**Comment by EvsChen** [comment]
I did some investigation and it seems to me a problem of `ts-json-schema-generator`. Currently it does not have support for `SyntaxKind = 165`, which is `FunctionType`.

So maybe the fix is add a node parser for `FunctionType` in the schema generator.

**Comment by kanitw** [comment]
Is FunctionType literally a function?  (JSON wouldn't support Function) 

I think the real culprit is `HTMLBodyElementEventMap` https://github.com/vega/vega-typings/blob/master/types/spec/stream.d.ts#L40. 

It might be better to redeclare the same keys. manually here to avoid the problems.

**Comment by EvsChen** [comment]
Yes, I guess that's a better solution, since we only need the keys from the event map.

BTW, from my point of view, `FunctionType` is a function, for example the type of handlers in `GlobalEventHandlers`.

**Comment by EvsChen** [comment]
I guess I should submit the PR to vega. Ignore this one.

**Comment by domoritz** [comment]
@EvsChen I fixed the schema generator so you can just import the type from Vega now.
